### PR TITLE
Rog corda 3159 fix

### PR DESCRIPTION
**Pull request by roger3cev**
See https://r3-cev.atlassian.net/browse/CORDA-3179 and https://r3-cev.atlassian.net/browse/CORDA-3159

### CODE REVIEW

File: workflows/src/main/kotlin/com/r3/corda/lib/tokens/workflows/flows/evolvable/UpdateEvolvableTokenFlow.kt [hunk 1]
```diff
@@ -29,6 +29,10 @@ class UpdateEvolvableTokenFlow(
 
     @Suspendable
     override fun call(): SignedTransaction {
+        require(ourIdentity in oldStateAndRef.state.data.participants) {
```
Comment (kasiastreich) [inline workflows/src/main/kotlin/com/r3/corda/lib/tokens/workflows/flows/evolvable/UpdateEvolvableTokenFlow.kt]: Check that it is in maintainers not participants?

File: workflows/src/main/kotlin/com/r3/corda/lib/tokens/workflows/flows/evolvable/UpdateEvolvableTokenFlowHandler.kt [hunk 2]
```diff
@@ -19,7 +20,13 @@ class UpdateEvolvableTokenFlowHandler(val otherSession: FlowSession) : FlowLogic
         if (notification.signatureRequired) {
             val signTransactionFlow = object : SignTransactionFlow(otherSession) {
                 override fun checkTransaction(stx: SignedTransaction) = requireThat {
-                    // TODO
+                    val ledgerTransaction = stx.toLedgerTransaction(serviceHub)
+                    val evolvableTokenTypeStateRef = ledgerTransaction.inRefsOfType<EvolvableTokenType>().single()
+                    val oldParticipants = evolvableTokenTypeStateRef.state.data.participants
```
Comment (kasiastreich) [inline workflows/src/main/kotlin/com/r3/corda/lib/tokens/workflows/flows/evolvable/UpdateEvolvableTokenFlowHandler.kt]: maintainer?

File: workflows/src/main/kotlin/com/r3/corda/lib/tokens/workflows/flows/evolvable/UpdateEvolvableTokenFlow.kt (hunk 1 above)
Comment (kasiastreich) [reply workflows/src/main/kotlin/com/r3/corda/lib/tokens/workflows/flows/evolvable/UpdateEvolvableTokenFlow.kt]: sorry, my bad, it seems it participants are maintainers

**Comment by roger3cev** [comment]
build

**Comment by roger3cev** [comment]
build

**Pull request merged**